Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -3660,7 +3660,9 @@ private static string ComputePipTimeoutDumpDirectory(ISandboxConfiguration sandb
             outputToLog = (standardOutEmpty ? string.Empty : standardOut) +
                 (!standardOutEmpty && !standardErrorEmpty ? Environment.NewLine : string.Empty) +
                 (standardErrorEmpty ? string.Empty : standardError);
-            pathsToLog = (standardOutEmpty ? string.Empty : standardOutPath) +
+            pathsToLog = m_sandboxConfig.OutputReportingMode == OutputReportingMode.FullOutputAlways

```

Comment:
>pathsToLog = m_sandboxConfig.OutputReportingMode == OutputReportingMode.FullOutputAlways [](start = 12, length = 88)

We should have some UTs for this output rendering. Please expand the set with a test case for this functionality.